app/connectors/ThirdPartyApplicationConnector.scala

```diff
@@ -242,6 +247,8 @@ private[connectors] object ThirdPartyApplicationConnectorDomain {
   case class TPAClientSecret(id: String, name: String, secret: Option[String], createdOn: DateTime, lastAccess: Option[DateTime])
 
   case class DeleteClientSecretRequest(actorEmailAddress: String)
+
+  case class UpdateIpAllowlistRequest(ipWhitelist: Set[String])
```

Comment:
There are places like this, where we can't use allowlist instead of whitelist because it's the name of the property we need to parse from TPA. Same with the TPA URL. We may have to fix this in the future.